test/master-playlist-controller.test.js

```diff
@@ -1127,18 +1127,26 @@ QUnit.test('calls to update cues on media when no master', function(assert) {
 
 QUnit.test('respects useCueTags option', function(assert) {
   let origHlsOptions = videojs.options.hls;
+  let hlsplaylistcuetags = 0;
```

Comment:
For this and other event counters, I think it is more readable with camelCase, e.g., `hlsPlaylistCueTags` or even `hlsPlaylistCueTagsEvents` to be more precise about what is being tracked